daemon/cmd/daemon_main.go

```diff
@@ -1598,9 +1600,8 @@ func checkHostFirewallWithEgressLB() {
 		(!option.Config.EnableHostServicesTCP || !option.Config.EnableHostServicesUDP ||
 			(option.Config.EnableExternalIPs && !netnsCookieSupport))
```

Comment:
For reviewers: Is there a simpler way to check that we are relying on neither kube-proxy nor `bpf_sock`? This seems rather cumbersome :/ (it's the direct translation of the check we have on BPF side).